Neos.Neos/Tests/Behavior/Features/Bootstrap/FeatureContext.php

```diff
@@ -144,15 +140,15 @@ public function theFollowingUsersExist(TableNode $table)
             }, Arrays::trimExplode(',', $row['roles']));
             $userService->createUser($row['username'], $row['password'], $row['firstname'], $row['lastname'], $roleIdentifiers);
         }
-        $this->getSubcontext('flow')->persistAll();
+        $this->persistAll();
     }
 
     /**
      * @Given /^I am authenticated with "([^"]*)" and "([^"]*)" for the backend$/
      */
     public function iAmAuthenticatedWithAndForTheBackend($username, $password)
     {
-        $this->visit('/neos/login');
+        $this->visit('https://t3n.de/');
```

Comment:
Ad injection? ;) But honestly, does it work?

Reply:
hahaha, lol, nope. The test's aren't executed at all cause they're tagged with @browser. It's a debug leftover where i wanted to test if selenium could handle absolute url's